### PR TITLE
Enable DataField, remove SASS from repo

### PR DESCRIPTION
**Pull request by kotielnikov**
### Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [ ] All new and existing tests passed

### Description

I've returned `DateField` and removed SASS from it

### CODE REVIEW

**Comment by kotielnikov** [comment]
@guiherzog now you can remove object-proxy from Pharos

File: package.json [hunk 1]
```diff
@@ -88,6 +91,7 @@
     "jest-puppeteer-docker": "^1.2.0",
     "jest-styled-components": "^6.3.1",
     "material-design-icons": "^3.0.1",
+    "node-sass": "^4.12.0",
```
Comment (dimabory) [inline package.json]: shouldn't it be removed?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "precise-ui",
-  "version": "0.7.1",
+  "version": "0.7.0",
```
Comment (dimabory) [inline package.json]:
```suggestion
  "version": "0.7.1",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,6 @@
 
 ## 0.7.1
 
-- Revert `DateField`
```
Comment (dimabory) [inline CHANGELOG.md]: Can you add a remark into `changelog` mentioning the issue with scss/sass?

**Pull request merged**